ludwig/features/image_feature.py

```diff
@@ -482,38 +563,46 @@ def __init__(self, input_feature_config: ImageInputFeatureConfig, encoder_obj=No
 
     def forward(self, inputs: torch.Tensor) -> torch.Tensor:
         assert isinstance(inputs, torch.Tensor)
-        assert inputs.dtype in [torch.uint8, torch.int64]
-
-        # casting and rescaling
-        inputs = inputs.type(torch.float32) / 255
+        assert inputs.dtype in [torch.float32]
 
         inputs_encoded = self.encoder_obj(inputs)
 
         return inputs_encoded
 
     @property
     def input_dtype(self):
-        return torch.uint8
+        return torch.float32
 
     @property
     def input_shape(self) -> torch.Size:
-        return torch.Size(
-            [self.encoder_obj.config.num_channels, self.encoder_obj.config.height, self.encoder_obj.config.width]
-        )
+        return torch.Size(self.encoder_obj.input_shape)
 
     @property
     def output_shape(self) -> torch.Size:
         return self.encoder_obj.output_shape
 
     @staticmethod
     def update_config_with_metadata(feature_config, feature_metadata, *args, **kwargs):
-        for key in ["height", "width", "num_channels", "scaling"]:
+        for key in ["height", "width", "num_channels", "standardize_image"]:
             setattr(feature_config.encoder, key, feature_metadata[PREPROCESSING][key])
 
+    @staticmethod
```

Comment:
I believe we got rid of this method. Defaults are sourced purely from schemas now. CC: @connor-mccorm @ksbrar

Reply:
We did. @connor-mccorm knows best, but you should be able to just remove this again, @jimthompson5802

Reply:
removed `populate_defaults()` method